src/lib_ccx/ccx_common_common.c

```diff
@@ -49,13 +49,13 @@ void millis_to_time(LLONG milli, unsigned *hours, unsigned *minutes,
 	unsigned *seconds, unsigned *ms)
 {
 	// LLONG milli = (LLONG) ((ccblock*1000)/29.97);
-	*ms = (unsigned)(milli % 1000); // milliseconds
+	*ms = (unsigned) milli % 1000; // milliseconds
```

Comment:
What's wrong with the parenthesis? They don't make the code slower, and readability is always better :-)

Reply:
Reverted, but IMO they actually decrease readability as you expect them to do something when they in fact don't do anything